src/server/handlers.js

```diff
@@ -0,0 +1,63 @@
+// -- required --------------------------
+const path = require('path');
+const fs = require('fs');
+
+
+// -- Error 404 -------------------------
+const errorHandler = (request, response) => {
+    response.writeHead(404, {'Content-Type' : 'text/html'});
+    response.end('<h1>Page Not Found : Error 404</h1>');
+};
+
+// -- Error 500 -------------------------
+const serverHandler = (request, response) => {
+    response.writeHead(505, {'Content-Type' : 'text/html'});
```

Comment:
505 is not the correct status code:
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_errors

Reply:
thank you. I did it.